primary/tests/nodes_bootstrapping_tests.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use itertools::Itertools;
+use std::{collections::HashMap, time::Duration};
+use telemetry_subscribers::TelemetryGuards;
+use test_utils::cluster::Cluster;
+use tracing::info;
+
+/// Nodes will be started in a staggered fashion. This is simulating
+/// a real world scenario where nodes across validators will not start
+/// in the same time.
+#[ignore]
```

Comment:
This is a long running test - as most of the ones that we introduced - and I believe they should stay out of our main pipeline. It can really slow down contribution/productivity if someone needs to wait at least 30 mins for tests to finish. I would consider though building a separate action that can run periodically and report the results and maybe block any merge if the last run wasn't successful